src/pages/CommandView/CommandView.tsx

```diff
@@ -1,86 +1,91 @@
-// import { Box } from '@mui/material'
-import useAxios from 'axios-hooks'
-// import { formBuilder } from 'builderForm/form_builder'
+import { Box } from '@mui/material'
 import Breadcrumbs from 'components/Breadcrumbs'
-// import CommandViewForm from 'components/command_view_form'
-import Divider from 'components/divider'
+import { Divider } from 'components/Divider'
+import { JobRequestCreationContext } from 'components/JobRequestCreation'
 import PageHeader from 'components/PageHeader'
 import { ServerConfigContainer } from 'containers/ConfigContainer'
-import { useEffect, useState } from 'react'
-// import ReactJson from 'react-json-view'
+import { CommandViewForm } from 'pages/CommandView/CommandViewForm'
+import { checkContext } from 'pages/CommandView/commandViewHelpers'
+import {
+  getJobSchema,
+  getModel,
+  getSchema,
+  getUiSchema,
+  getValidator,
+} from 'pages/CommandView/form-data'
+import { useContext } from 'react'
+import ReactJson from 'react-json-view'
 import { useParams } from 'react-router-dom'
-import { Command, System } from 'types/backend-types'
+import { AugmentedCommand, StrippedSystem } from 'types/custom-types'
 
 const CommandView = () => {
-  const { authEnabled } = ServerConfigContainer.useContainer()
-  const [systems, setSystems] = useState<System[]>([])
-  const {
+  const { debugEnabled } = ServerConfigContainer.useContainer()
+  const { namespace, systemName, version, commandName } = useParams()
+  const context = useContext(JobRequestCreationContext)
+  const { system, command, isJob } = context
+
+  const checkedParams = checkContext(
```

Comment:
This will make sure all the relevant information from the context is available. It returns `null` if everything's OK and a component that navigates away otherwise.